tests/merchStore.test.ts

```diff
@@ -406,3 +409,274 @@ describe('merch item options', () => {
       .toStrictEqual(item.options[0].getPublicMerchItemOption(true));
   });
 });
+
+describe('merch order pickup events', () => {
+  test('GET /order/pickup/future returns pickup events that haven\'t ended yet', async () => {
+    const conn = await DatabaseConnection.get();
+    const admin = UserFactory.fake({ accessType: UserAccessType.ADMIN });
+    const pastPickupEvent = MerchFactory.fakeOrderPickupEvent({
+      start: moment().subtract(2, 'hour').toDate(),
+      end: moment().subtract(1, 'hour').toDate(),
```

Comment:
We already have a few pickup event tests defined in `merchOrder.test.ts` - some of which overlap with the ones written here. Can we move these tests to `merchOrder.test.ts` and merge the duplicated tests? `merchOrder.test.ts` is meant for testing constructs related to merch orders (i.e. orders, pickup events, fulfillment) whereas `merchStore.test.ts` is meant for testing the store structures like collections, items, and options. Apologies if that was unclear, I can add some comments within those test files to make the distinction clearer

Reply:
This PR was made before your refactor, hence the duplication. I’ll merge the tests.

Reply:
Resolved in `c38ac89`.